client/extensions/woocommerce/app/store-stats/index.js

```diff
@@ -117,24 +99,35 @@ class StoreStats extends Component {
 					/>
 				</StatsPeriodNavigation>
 				<div className="store-stats__widgets">
-					<div className="store-stats__widgets-column spark-widgets" key="sparkwidgets">
-						<Module
-							siteId={ siteId }
-							header={ null }
-							emptyMessage={ translate( 'No data found.' ) }
-							query={ Object.assign( {}, ordersQuery, { date: unitQueryDate } ) }
-							statType="statsOrders"
-						>
-							{ widgetList1 }
-							{ widgetList2 }
-						</Module>
-					</div>
+					{ sparkWidgets.map( ( widget, index ) => (
+						<div className="store-stats__widgets-column spark-widgets" key={ index }>
+							<Module
+								siteId={ siteId }
+								emptyMessage={ translate( 'No data found.' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **22** times:
`translate( 'No data found' )` *ES Score: 10*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/fix/WCS-one-query-per-module)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).